joomla/joomla-cms
# Removing the notice when a global value is not found.

Pull Request for Issue #12782.

### Summary of Changes
Removing the notice when the global value isn't found (not yet saved) as it is more confusing to users than it helps.

### Testing Instructions
On a new install, create a menu item for com_content or edit an article.
You should see a notice that no global value was found. Reason is that either component settings, com_menu settings or global settings aren't saved yet.
After PR this message no longer appears.

### Documentation Changes Required
None
